src/engine/LocalVocab.h

```diff
@@ -107,25 +107,24 @@ class LocalVocab {
     // Also merge the `vocabs` `LocalBlankNodeManager`s, if they exist.
     using LocalBlankNodeManager =
         ad_utility::BlankNodeManager::LocalBlankNodeManager;
-    using sharedLbnm = std::shared_ptr<const LocalBlankNodeManager>;
+    using sharedManager = std::shared_ptr<const LocalBlankNodeManager>;
 
-    auto lbnmView =
-        vocabs |
-        std::views::transform([](const LocalVocab& vocab) -> const sharedLbnm {
-          return vocab.localBlankNodeManager_;
-        });
+    auto localManagersView =
+        vocabs | std::views::transform(
+                     [](const LocalVocab& vocab) -> const sharedManager {
```

Comment:
```suggestion
                     [](const LocalVocab& vocab) -> const auto& {
```
I still want to avoid the unnecessary copies of shared pointers.